src/daemon/custom_image_host.h

```diff
@@ -20,24 +20,46 @@
 
 #include <multipass/vm_image_host.h>
 
+#include <QString>
+
+#include <memory>
+#include <string>
 #include <unordered_map>
+#include <vector>
 
 namespace multipass
 {
+constexpr auto no_remote = "";
+constexpr auto snapcraft_remote = "snapcraft";
+
```

Comment:
Looks like they are only used in custom_image_host.cpp so perhaps to avoid polluting multipass namespace, define them there instead?

Reply:
Yep, sure.